visualizer/package.json

```
@@ -36,7 +36,7 @@
     "react-dom": "^17.0.1",
     "react-dropzone": "^11.4.0",
     "react-icons": "^4.2.0",
-    "react-router-dom": "^5.2.0",
+    "react-router-dom": "6",
```

```suggestion
    "react-router-dom": "^6.0.0",
```

I think it's better to make sure we are able to build with patch releases.

Also, is there any dependency on the react version itself and react-router v6?

---

It requires React >=16.8 which introduced that and our React version dependency is already higher. I think react-router-dom is react-router for the browser and there's also react-router-native for react native. The [documentation says to install react-router-dom](https://reactrouter.com/docs/en/v6/getting-started/overview)